docker/package_managers/install_pkgs.bzl

```diff
@@ -167,8 +171,7 @@ _attrs = {
         default = "",
     ),
     "output_image_name": attr.string(
-        doc = ("Name of container_image produced with the packages installed."),
-        mandatory = True,
+        doc = ("Name of container_image produced with the packages installed. If not specified, resolves to the current target target label. *Must be unique* across the entire build tree."),
```

Comment:
nit: not your fault, but why is this string wrapped in `()`?

Reply:
Ah this is actually my bad, looking at other rules it's to allow breaking the help text over multiple lines. Let me amend real quick

Reply:
Fixed the formatting to match the other rules. LMK if that's okay to resolve now 👍